web/src/components/ModalZeroButton.tsx

```diff
@@ -2,19 +2,33 @@ import { Heading } from "@/components/njwds-extended/Heading";
 import { Icon } from "@/components/njwds/Icon";
 import { ContextualInfoContext } from "@/contexts/contextualInfoContext";
 import { Dialog, DialogContent, DialogTitle, IconButton } from "@mui/material";
-import { ReactElement, ReactNode, useContext } from "react";
+import { ReactElement, ReactNode, useContext, useEffect, useRef } from "react";
 
 interface Props {
   isOpen: boolean;
   close: () => void;
   title: string;
   children: ReactNode;
   unpaddedChildren?: ReactNode;
-  uncloseable?: boolean;
 }
 
 export const ModalZeroButton = (props: Props): ReactElement => {
   const { contextualInfo } = useContext(ContextualInfoContext);
+
+  const dialogRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (dialogRef.current) {
+      const firstFocusableElement = dialogRef.current.querySelector<HTMLElement>(
+        'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])'
+      );
+      if (firstFocusableElement) {
+        firstFocusableElement.focus();
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [dialogRef.current]);
```

Comment:
Forgot to check if this breaks your change in some way though! Stay tuned.

Reply:
Well running NVDA locally is more complicated than I thought. @adidner any recommendations?

Reply:
Use voice over? I only know how to run NVDA on a windows machine and was going to test after when it's on dev (issues with the testing site)

But honestly just getting it to focus on the close button when the dialog appears should make everything work for all the screen readers. 

And also I'm pretty sure that won't work because the ref will be null if you only run it on Mount. But would love to be proven wrong.

Reply:
You're totally right. After doing some research - the ref itself changing won't trigger a rerender but the deps are still evaluated when something else causes a rerender and the ref value is different. Since deps are there as listeners for the effect to rerun, this approach is technically an anti-pattern in React and could cause some bugs in the future but I'm not too concerned with that since MUI is going away soon anyway.